parkingPlacesRecorder.py

```diff
@@ -0,0 +1,48 @@
+import cv2
+import json
+import argparse
+from sys import exit
+from datetime import datetime
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--img_dir', type=str, default='./photos/',
+                    help='directory where camera images should be stored')
+parser.add_argument('-pp', '--parking_places', type=str, default='./parkingPlaces.json',
+                    help='json file with parking places locations')
+parser.add_argument('-cs', '--camera_source', default=0,
+                    help='example: rtsp://username:password@192.168.1.64/1')
+
+
+def load_json(file_path):
+    places = {}
+
+    try:
+        with open(file_path, 'r') as file:
+            places = json.load(file)
+    except FileNotFoundError:
+        exit("File " + file_path + " not found")
+
+    return places
+
+
+if __name__ == '__main__':
+    args = parser.parse_args()
+
+    # 1. Take a picture from camera
+    cap = cv2.VideoCapture(args.camera_source)
+    ret, frame = cap.read() if args.camera_source != 0 else (True, cv2.imread("./image4.png"))
```

Comment:
proč dávat jméno souboru přímo do kódu? Lepší by bylo to dát jako argument s defaultní hodnotou, a nebo kontrolovat jestli není specifikována, pokud ne, vyhodit vyjímku že není specifikován žadný zdroj